.github/workflows/acceptance_tests_common.yml

```diff
@@ -34,101 +37,115 @@ jobs:
+      - name: Install Podman
         run: sudo apt install podman=3.4.4+ds1-1ubuntu1 --allow-downgrades
-      - name: welcome_message
+      - name: Install Python dependencies
+        run: pip install pyyaml
+      - name: Welcome message
         run: echo "Running acceptance tests. More info at https://github.com/uyuni-project/uyuni/wiki/Running-Acceptance-Tests-at-PR"
       - uses: actions/checkout@v4
-      - name: Cache-jar-files
+      - name: Cache jar files
         uses: actions/cache@v4
         with:
           path: java/buildconf/ivy/repository/
           key: ${{ runner.os }}-build-cache-uyuni-jars-${{ hashFiles('**/java/buildconf/ivy/*.xml') }}
           restore-keys: |
             ${{ runner.os }}-build-cache-uyuni-jars-
-
-      - name: Cache-obs-to-maven files
+      - name: Cache obs-to-maven files
         uses: actions/cache@v4
         with:
           path: java/.obs-to-maven-cache
           key: ${{ runner.os }}-build-cache-uyuni-obs-to-maven-${{ hashFiles('**/java/buildconf/ivy/obs-maven-config.yaml') }}
           restore-keys: |
             ${{ runner.os }}-build-cache-uyuni-obs-to-maven-
-      - name: Cache-nodejs
+      - name: Cache NodeJS modules
         uses: actions/cache@v4
         with:
           path: web/html/src/node_modules
           key: ${{ runner.os }}-build-cache-uyuni-nodejs-${{ hashFiles('**/web/html/src/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-build-cache-uyuni-nodejs-
-      - name: create_tmp
+      - name: Create temporary directories
         run: ./testsuite/podman_runner/01_setup_tmp_dirs.sh
-      - name: create-podman-network
+      - name: Create Podman network
         run: ./testsuite/podman_runner/02_setup_network.sh
-      - name: start_controller_and_registry_and_buildhost
+      - name: Start controller, registry and build host
         run: ./testsuite/podman_runner/03_run_controller_and_registry_and_buildhost.sh 
-      - name: create_ssh_conf
+      - name: Create SSH configuration in controller
         run: ./testsuite/podman_runner/04_setup_ssh_controller.sh 
-      - name: install_gems_in_controller
+      - name: Install gems in controller
         run: ./testsuite/podman_runner/05_install_gems_in_controller.sh
-      - name: collect_and_tag_flaky_tests_in_controller
+      - name: Parse recommended tests coming from the input as XML
+        if: ${{ inputs.recommended_tests != '' }}
+        run: python .github/scripts/get_tests.py '${{ inputs.recommended_tests }}' > testsuite/run_sets/filter.yml
+      - name: Generate recommended tests as YAML
+        if: ${{ inputs.recommended_tests != '' }}
+        run: |
+          ./testsuite/podman_runner/19_generate_recommended_tests_yml.sh
+          TESTS_COUNT=$(wc -l < /testsuite/run_sets/recommended_tests.yml)
+          if [ "$ROW_COUNT" -eq 0 ] || [ "$ROW_COUNT" -gt 30 ]; then
+            echo "⚠️ WARNING: $ROW_COUNT recommended tests found. Running the full test suite instead."
```

Comment:
I would change "Running the full test suite instead" by "Look at the results of the full test suite, instead".

Reply:
Other than this minor issue. LTGM